service/src/integrationTest/kotlin/fi/espoo/evaka/varda/VardaClientIntegrationFailTest.kt

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.varda
+
+import com.github.kittinunf.result.Result
+import fi.espoo.evaka.FullApplicationTest
+import org.junit.jupiter.api.Assertions.fail
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+
+class VardaClientIntegrationFailTest : FullApplicationTest() {
+    private lateinit var httpsServer: VardaClientIntegrationMockHttpsServer
+
+    @BeforeAll
+    fun initDependencies() {
+        httpsServer = VardaClientIntegrationMockHttpsServer.start()
```

Comment:
It's not critical, but we should probably stop the server by calling `httpsServer.close()` in an `AfterAll`-annotated function. I noticed this is also missing in the existing Koski tests where a mock server is used.

Reply:
Changed.